### PR TITLE
Kanna Paper command

### LINKED ISSUE #692: Kanna Paper Meme Command
`s!kanna <text>`

image

### PR DESCRIPTION
**Pull request by Tsugami**
Closes #692

### CODE REVIEW

File: src/utils/CanvasTemplates.js [hunk 1]
```diff
@@ -803,4 +803,77 @@ module.exports = class CanvasTemplates {
...
+
+      return lines
+    }
+
+    ctx.drawImage(background, 0, 0, background.width, background.height)
+    ctx.font = '15px "Arial"'
+    ctx.rotate(12 * Math.PI / 180)
+
+    const lines = lineBreak(text, 13, 6)
+    console.log(lines)
```
Comment (jaxxibae) [inline src/utils/CanvasTemplates.js:874]:
```suggestion

```

**Review by jaxxibae — DISMISSED** [review]

File: src/locales/en-US/commands.json [hunk 2]
```diff
@@ -1701,5 +1701,10 @@
     "verified": "Verified account",
     "yes": "Yes",
     "no": "No"
+  },
+  "kannapaper": {
+    "commandDescription": "Generates a text on the kanna paper.",
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json:1706]:
```suggestion
    "commandDescription": "Generates an image of Kanna (from Miss Kobayashi's Dragon Maid) holding a paper with text",
```

File: src/locales/en-US/commands.json [hunk 3]
```diff
@@ -1701,5 +1701,10 @@
...
+    "commandUsage": "<text>",
+    "missingText": "You have to give me some text for the kanna paper!"
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json:1708]:
```suggestion
    "missingText": "You have to give me some text for the paper!"
```

**Review by pedrofracassi — CHANGES_REQUESTED** [review]

**Review by pedrofracassi — APPROVED** [review]
image

**Review by Doges — APPROVED** [review]
LGTM




nice

**Review by almeidx — APPROVED** [review]

**Pull request merged**